[2022-02-03 20:53:12,900] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:13,176] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:13,180] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:13,184] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:53:13,187] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:13,459] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-01 00:00:00+00:00
[2022-02-03 20:53:13,516] {standard_task_runner.py:52} INFO - Started process 6722 to run task
[2022-02-03 20:53:13,540] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpj9uj485c', '--error-file', '/tmp/tmpdwd8t07s']
[2022-02-03 20:53:13,550] {standard_task_runner.py:77} INFO - Job 37: Subtask download_dataset_task
[2022-02-03 20:53:14,188] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:53:14,721] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-02-03 20:53:14,735] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:53:14,753] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:53:14,819] {subprocess.py:85} INFO - Output:
[2022-02-03 20:53:14,898] {subprocess.py:89} INFO - /tmp/***tmpbwc0_9i7
[2022-02-03 20:53:14,917] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:53:17,310] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20190301T000000, start_date=20220203T205312, end_date=20220203T205317
[2022-02-03 20:53:17,601] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:53:18,088] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
